tests/Ordering.UnitTests/Application/SetStockRejectedOrderStatusCommandTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+
+namespace eShop.Ordering.UnitTests.Application;
+
+public class SetStockRejectedOrderStatusCommandTest
+{
+    [Fact]
+    public void Set_Stock_Rejected_OrderStatusCommand_Check_Serialization()
+    {
+        // Arrange
+        var command = new SetStockRejectedOrderStatusCommand(123, new List<int> { 1, 2, 3 });
+
+        // Act
+        var json = JsonConvert.SerializeObject(command);
+        var deserializedCommand = JsonConvert.DeserializeObject<SetStockRejectedOrderStatusCommand>(json);
```

Comment:
Why is this test using JSON.NET?

Reply:
Updated the code to use System.Text.Json.JsonSerializer instead of JSON.NET.